src/clients/inst_root.rb

```diff
@@ -168,18 +162,8 @@ def main
         )
       )
 
-      if @check_CA_constraints
-        @helptext = Ops.add(
-          @helptext,
-          Builtins.sformat(
-            # additional help text about password
-            _(
-              "<p>If you intend to use this password for creating certificates,\nit has to be at least %1 characters long.</p>"
-            ),
-            @pw_min_CA
-          )
-        )
-      end
+      validator = Users::CAPasswordValidator.new
+      @helptext << validator.help_text if validator.enabled?
```

Comment:
I also considered it. I don't have a strong preference for any of the options but in the end this approach somehow felt more flexible and explicit to me. `help_text` does always what is expected (returning a help text) and is the dialog who decides if it's worth adding it to the big help text or not.

Intentionally adding an empty string to the big text help sounds a little bit more tricky. Doing something non-obvious for the sake of encapsulation.